lib/npm.js

```diff
@@ -329,6 +341,14 @@ class Npm extends EventEmitter {
     return this.#logFile.files
   }
 
+  get logsDir () {
+    return this.config.get('logs-dir') || join(this.cache, '_logs')
```

Comment:
Can we default this like `cache`?

Reply:
I couldn't get that to work nicely with how we do config definitions. I tried setting the `defaults: join(cache, '_logs')` in the definitions, but then if you pass in `--cache=./my-cache`, that value isn't populated until after `config.load()`. So I don't think we can do that until we have a truly derived config system.

Reply:
Ah right, that only works for the flattened versions.